src/frontends/ir/src/input_model.cpp

```diff
@@ -4,23 +4,24 @@
 
 #include "input_model.hpp"
 
-#include <xml_parse_utils.h>
-
-#include <ir_deserializer.hpp>
-#include <ngraph/opsets/opset1.hpp>
-#include <openvino/op/util/framework_node.hpp>
-#include <pugixml.hpp>
-
+#include "ir_deserializer.hpp"
+#include "openvino/core/except.hpp"
 #include "openvino/core/validation_util.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/op/util/variable.hpp"
 #include "openvino/opsets/opset.hpp"
+#include "openvino/util/common_util.hpp"
+#include "pugixml.hpp"
+#include "xml_parse_utils.h"
 
-using namespace ngraph;
-using namespace InferenceEngine;
-
+OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
what to do with `ngraph::runtime::AlignedBuffer` ?

Reply:
We need to move it openvino developer API